membership-common/src/main/scala/com/gu/memsub/subsv2/Catalog.scala

```diff
@@ -0,0 +1,17 @@
+package com.gu.memsub.subsv2
+
+import com.gu.config.SubsV2ProductIds.ProductIds
+import com.gu.memsub.Subscription.{ProductRatePlanChargeId, ProductRatePlanId}
+import com.gu.memsub.subsv2.Catalog.CatalogMap
+
+object Catalog {
+
+  type CatalogMap = Map[ProductRatePlanId, ProductRatePlan]
+
+  // dummy ids for stripe (non zuora) products
+  val guardianPatronProductRatePlanId: ProductRatePlanId = ProductRatePlanId("guardian_patron")
+  val guardianPatronProductRatePlanChargeId: ProductRatePlanChargeId = ProductRatePlanChargeId("guardian_patron")
```

Comment:
now that we pass the catalog around everywhere to interpret the low level zuora ids, we need to add a fake entry for patron stripe subscriptions, so that they can be looked up by the controllers.